testbed/tests/assertions.py

```diff
@@ -11,6 +13,20 @@ def assert_set_get(obj, name, value):
 
 
 def assert_color(actual, expected):
-    for component in ["r", "g", "b"]:
-        assert getattr(actual, component) == getattr(expected, component)
-    assert actual.a == approx(expected.a, abs=(1 / 255))
+    if expected is None:
+        if actual is None:
+            return
+        else:
+            fail(f"Expected TRANSPARENT, got {actual}")
+    elif expected == TRANSPARENT:
+        if actual == TRANSPARENT:
+            return
+        else:
+            fail(f"Expected TRANSPARENT, got {actual}")
+    else:
+        if actual is None or actual == TRANSPARENT:
+            fail(f"Expected {expected}, got {actual}")
```

Comment:
Huh... I worked myself into a corner trying to avoid an `x == None` comparison (which flake8 reports as E711); but it looks like the `in` comparison and the assertion comparison doesn't trigger that.